package.json

```diff
@@ -160,7 +161,9 @@
   },
   "resolutions": {
     "@types/react": "^16.14.26",
-    "@types/react-dom": "^16.9.16"
+    "@types/react-dom": "^16.9.16",
+    "@babel/traverse": "^7.24.7",
```

Comment:
Why was this needed?

Reply:
some transitive instances are lagging in @storybook/react-webpack5 and jest

![Screenshot 2024-06-18 at 1 48 07 PM](https://github.com/appfolio/react-gears/assets/26424381/8c962ae6-a55a-4eb8-84ad-4bf2b2044b6c)